content/v1/deployment/helm/drivers/upgrade/powerstore.md

```diff
@@ -7,6 +7,10 @@ weight: 1
 Description: Upgrade PowerStore CSI driver
 ---
 
+{{% pageinfo color="primary" %}}
+An issue impacting usage of the NVMe protocol has been identified in the PowerStore CSI Driver version v2.11.0. The problem prevents mounts of NVMe volumes across all orchestration platforms. Users are advised to use an alternate block protocol (e.g. iSCSI, FC) or delay upgrading to the v2.11.0 version of the PowerStore driver until an updated driver has been published.
```

Comment:
This upgrade page is in the context of upgrading 2.9 to 2.10.1, I think we can leave out the banner for 2.11 here.

Reply:
Removed.